mc-to-mt-tp-conv/conv.sh

```diff
@@ -1,31 +1,31 @@
 #!/bin/bash
-cd "$(dirname "$0")"
-cd ..
+cd -- "$(dirname "$0")" || exit 1
+cd .. || exit 1
 rm -rf minetest-texture-pack
 rm -rf uctextures
 mkdir textures
 mkdir uctextures
-cd uctextures
+cd uctextures || exit 1
 mkdir b
 mkdir i
-cd ..
+cd .. || exit 1
 
 
-FILES=$(dirname "$0")/../assets/minecraft/textures/blocks/*.png
+FILES="$(dirname "$0")/../assets/minecraft/textures/blocks/*.png"
 for f in $FILES
```

Comment:
In `for f in $FILES`, it splits by spaces and expands the *, but it should only expand the *. I suggest to remove the FILES variable and adjust the loop head:
```suggestion
for f in "$(dirname "$0")"/../assets/minecraft/textures/blocks/*.png
```

Example code for illustration:
```bash
mkdir "foldername with space"
:> "foldername with space/file.txt"
ls -R
x="./foldername with space/*.txt"
for f in $x; do
	echo "Loop 1, f is: $f"
done
for f in "./foldername with space/"*.txt; do
	echo "Loop 2, f is: $f"
done
```

Output from the loops:
```
Loop 1, f is: ./foldername
Loop 1, f is: with
Loop 1, f is: space/*.txt
Loop 2, f is: ./foldername with space/file.txt
```